statler_site/templates/play-list-page.html

```diff
@@ -1,61 +1,352 @@
+   }
+.ribbon-stitches-top {
+   margin-top:2px;
+   border-top: 1px dashed rgba(0, 0, 0, 0.2);
+   -moz-box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.5);
+   -webkit-box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.5);
+   box-shadow: 0px 0px 2px rgba(255, 255, 255, 0.5);
+   }
+.ribbon-stitches-bottom {
+   margin-bottom:2px;
+   border-top: 1px dashed rgba(0, 0, 0, 0.2);
+   -moz-box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.3);
+   -webkit-box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.3);
+   box-shadow: 0px 0px 2px rgba(255, 255, 255, 0.3);
+   }
+
+
+</style>
+              <script  type="text/javascript">
+$(document).ready(function() {
+    var text_max = 500;
+    $('#textarea_feedback').html(text_max + ' characters remaining');
+
+    $('#textarea').keyup(function() {
+        var text_length = $('#textarea').val().length;
+        var text_remaining = text_max - text_length;
 
+        $('#textarea_feedback').html(text_remaining + ' characters remaining');
+    });
+});
+
+</script> <script>
+
+      // This script makes an API call and displays the returned data
       // This string will be filled out by Django when the page is served.
       // It contains the URL to make an API call to
-      var apiEndpointUrl = '{% url 'api:play-list' %}';
-
+      var apiEndpointUrl = '{% url 'api:play-detail' play_id %}';
       // jQuery code normally starts with the .ready function
       $(document).ready(function(){
         // Get JSON from a URL as an object
         $.getJSON(apiEndpointUrl, function(result){
           // display the returned json dynamically
-          $.each(result, function(index, value){
-
-            // this indentation helps because it looks a bit like html
-            $('#play-list').append(
-              //the whole list item is a link to the play description page
-              $('<a/>', {href: 'play/'+value.url_title}).append(
-                $('<li/>').append(
-                  $('<h1/>').append(value.title),
-                  $('<p/>').append(value.description),
-                  $('<p/>').append(
-                    'Directed by: ', value.director,
-                    // &#x2022 is a bullet, which makes a nice separator
-                    ' &#x2022; Actors: ', value.actors,
-                    ' &#x2022; Showtimes: ', value.show_times
-                  )
-                )
-              )
+          // fill in page title
+          $(".header > h1").append(result.title);
+          // fill in description
+          $('#description').append(
+            $("<p/>").append(result.description)
+          );
+          $('#details').append(
+            $("<p/>").append("Directed by: ", result.director),
+            $("<p/>").append("Actors: ", result.actors),
+            $("<p/>").append("Showtimes: ", result.show_times)
+          );
+          // fill in reviews
+          $.each(result.reviews, function(index, review) {
+            $("#review-list").append(
+              $("<li/>").append(review.text)
```

Comment:
It looks like all the code in the play list page has been replaced with the code from the play details page.